internal/scm/github/github.go

```diff
@@ -357,6 +358,31 @@ func (g *Github) GetPullRequests(ctx context.Context, branchName string) ([]scm.
 		return nil, err
 	}
 
+	wg, ctx := errgroup.WithContext(ctx)
```

Comment:
Haven't used errgroup before, just waitgroups. This seems very handy :)

Reply:
Yes I always use it instead waitgroup, its a nice wrapper lib :)